cassiopeia/plotting/local.py

```diff
@@ -0,0 +1,829 @@
+import warnings
+from collections import defaultdict
+from typing import Dict, List, Optional, Tuple, Union
+
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import plotly.graph_objects as go
+from matplotlib.colors import hsv_to_rgb
+from typing_extensions import Literal
```

Comment:
should this go above with the other `typing` imports?

Reply:
I'm not sure which way we prefer. `typing_extensions` is technically a 3rd party library, so it should come after native Python libraries? This is also what is done in `itol_utilities.py`

Reply:
Yeah that sounds reasonable, thanks for clarifying!